SOP/SOP-operator.adoc

```diff
@@ -0,0 +1,69 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:table-caption!:
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= UnifiedPush Operator - Standard Operating Procedures
+
+:toc:
+toc::[]
+
+== UnifiedpushOperatorDown
+
+Troubleshoot using the following steps via either the console or the cli:
+
+=== Console:
+
+. Log into the OpenShift console
+. Switch to the UnifiedPush namespace
+. Go to `Deployments -> unifiedpush-operator >> Events` and check if was possible pull the image
+. Check the logs of the pod for any errors
+.. Navigate to `Applications -> Pods -> unifiedpush-operator-<xyz123> -> Logs`
+.. Check for any errors in the logs
+. Try to scale up the pod if it is not present
+.. Navigate to `Overview -> Click Dropdown for Deployment unifiedpush-operator -> Click on up arrow to scale the pod to 1`
+
+=== CLI:
+
+. Login
+.. Run `oc login <openshift-url>:8443 -u <username> -p <password>`
+. Switch to the UnifiedPush namespace
+.. Run `oc project unifiedpush-operator`
+. If the operator pod is present check the logs of the pod for any errors
+. Get the operator pod name -> `oc get pods | grep operator`
+. `oc logs <operator-podname>`
+. Save the logs by running `oc logs <operator-podname> > <filename>.log`
+. If the operator pod is not preset scale up the pod
+.. `oc scale deployments unifiedpush-operator --replicas=1`
```

Comment:
This might need some update later as @secondsun is doing some work around scaling ups, but for now this is good.

Reply:
Sure .. if he changed it we can create a PR to update this part after.